bin/disco_aws.py

```diff
@@ -226,6 +226,36 @@ def get_preferred_private_ip(instance):
         return interfaces[1].private_ip_address
 
 
+def format_listhosts_header(args, most):
+    line = u"{0:<11} {1:<25} {2:<15}".format('ID', 'Hostclass', 'IP')
```

Comment:
Using tabulate is even better as you don't need to pre-format the strings. See https://github.com/amplifylitco/asiaq/pull/141/files